content/knowledge_base/consensus/avalanche/specification.md

```diff
@@ -0,0 +1,38 @@
+# Avalanche
+
+An internet-scale electronic payment system, evaluated in a large scale deployment, constructed with Snow family of algorithms. Avalance's tested confirmation latency goes down to 1.35 sec. System's bottleneck is transaction verification.
```

Comment:
```suggestion
An internet-scale electronic payment system, evaluated in a large scale deployment, based on the Snow family of algorithms. Avalanche's tested confirmation latency goes down to 1.35 sec. The system's bottleneck is transaction verification.
```

I'm curious, where is this 1.35s metric coming from?

Reply:
Seems like I need to elaborate on this,

Reply:
The text changes are added https://github.com/optakt/optakt.github.io/pull/13/commits/0a1cb902f3417c3bc20f9197df71d3d65a1b6421#diff-69f45991c0732c6f5ff297a5a86ffa91662d50369fc8057b3875361f5b80a2fcR3-R7
I removed the 1.35ms reference because it is a result from a test of the Avalanche network that has a quite long description for the initial specifications and its place is not here. Its from the white paper, I plan to add it in a conclusion section lately